tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
-minversion=2.3.1
-envlist = py36,py37,py38,py39,py310,py311,black,pyanalyze
+minversion=4.0.8
```

Comment:
Not really, I just bumped it to what I currently had installed - though that was partly while trying to work out other issues and don't think this mattered at all in the end. Didn't realize tox 4.0 was only a week old so I suppose it might make sense to not bump the minversion all the way there, but it's what's default installed with pip anyway.

If there's no reason for any particular version, you could just drop that config line entirely.

Reply:
Reverted that change back to 2.3.1